README.md

```diff
@@ -1 +1,947 @@
-# securing-restful-services-with-ldap
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/securing-restful-services-with-ldap.svg?branch=master)](https://travis-ci.org/ballerina-guides/securing-restful-services-with-basic-auth)
+
+# Securing RESTful Services with LDAP
+
+In this guide you will learn about integrating external LDAP to a service that enforces authentication and 
+authorization checks using Ballerina.
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Developing the service](#developing-the-service)
+- [Testing](#testing)
+- [Deployment](#deployment)
+- [Observability](#observability)
+
+## What you’ll build
+To understanding how you can secure a RESTful web service using Ballerina, let’s continue to secure the web service you created in "RESTful Service" Ballerina by Guide.
```

Comment:
Change "To understand how you can secure a RESTful web service using Ballerina, let’s secure the web service you created in the [RESTful Service guide](https://ballerina.io/learn/by-guide/restful-service/)."